xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -38,7 +38,8 @@ import (
 	"google.golang.org/grpc/xds/internal/xdsclient"
 )
 
-const edsName = "eds_experimental"
+// Name is the name of the cluster_resolver balancer.
+const Name = "eds_experimental"
```

Comment:
🤯

Reply:
Yeah, I know.

I also changed CDS to use this exported var, instead of hardcoding `eds_experimental`.
Hope that makes things _less weird_